src/util/fields.py

```diff
@@ -1070,22 +1193,9 @@ def _update_field_usage_multi(cr, models, old, new, domain_adapter=None, skip_in
         """
         cr.execute(q.format(col_prefix=col_prefix), p)
 
-        # ir.exports.line
-        q = """
-            UPDATE ir_exports_line l
-               SET name = regexp_replace(l.name, %(old)s, %(new)s, 'g')
-        """
+        # ir.exports
         if only_models:
-            q += """
-              FROM ir_exports e
-             WHERE e.id = l.export_id
-               AND e.resource IN %(models)s
-               AND
-            """
-        else:
-            q += "WHERE "
-        q += "l.name ~ %(old)s"
-        cr.execute(q, p)
+            rename_ir_exports_fields(cr, {model: {old: new} for model in only_models})
```

Comment:
Inlining the function
* No need to build extra dicts
* Can use jsonb trick to simplity the query (instead of values)
```suggestion
            # Rename ir.exports fields
            matching_exports = _get_resolved_ir_exports(
                cr, fields=[(model, old) for model only_models]
            )
            if matching_exports:                
                _logger.debug("Renaming %d export template lines with renamed fields", len(matching_exports))
                fixed_lines_paths = {}
                for line_id, resolved_paths in matching_exports.items():
                    for path_part in resolved_paths:                        
                        fixed_path = fixed_lines_paths.get(line_id, list(path_part.path))
                        fixed_path[path_part.part_index - 1] = new
                        fixed_lines_paths[line_id] = fixed_path
                cr.execute(
                    """
                    UPDATE ir_exports_line
                       SET name = (%s::jsonb)->>(id::text)
                     WHERE id IN %s
                    """,
                    [Json(fixed_lines_paths), tuple(fixed_lines_paths)],
                )
```